### ISSUE TITLE
could you please explain the rational behind applying clip on ranking loss 

### ISSUE THREAD

**Issue by seganrasan** [raphaelsty/neural-cherche#9]
https://github.com/raphaelsty/neural-cherche/blob/97ba18d31a18ca41bbdc1e3d6a3f17769f3e1fff/neural_cherche/losses/ranking.py#L74

**Comment by raphaelsty** [comment]
Hi, I've observed that digital instability can occur with the loss I've set up, I'm considering replacing this loss with [torch.nn.CrossEntropyLoss(https://pytorch.org/docs/stable/generated/torch.nn.CrossEntropyLoss.html#torch.nn.CrossEntropyLoss). Any contribution to this is welcome.

**Comment by raphaelsty** [comment]
Gradient is no longer clipper, neural-cherche 1.1.0 add cross-entropy as ranking loss function :)
